anoncreds/tests/anoncreds_demos.rs

```diff
@@ -443,21 +451,18 @@ fn anoncreds_with_revocation_works_for_single_issuer_single_prover() {
         &cred_defs,
         &pres_request,
         &prover_wallet,
-        Some(prover_timestamp),
+        Some(time_after_creating_cred),
         Some(&rev_state),
     );
 
     // Verifier verifies presentation of not Revoked rev_state
-    // TODO: rev reg def id is the same as the rev reg id?
-    let rev_reg_def_id = RevocationRegistryDefinitionId::new_unchecked(REV_REG_ID);
     let rev_reg_def_map = HashMap::from([(&rev_reg_def_id, &rev_reg_def_pub)]);
-
-    // Create the map that contines the registries
-    let rev_timestamp_map = HashMap::from([(prover_timestamp, &cred_rev_reg)]);
-
-    let rev_reg_id = RevocationRegistryId::new_unchecked(REV_REG_ID);
+    // Create the map that containes the registries
+    let rev_reg =
+        <&RevocationStatusList as Into<Option<RevocationRegistry>>>::into(&issued_rev_status_list)
+            .unwrap();
```

Comment:
```suggestion
    let rev_reg: Option<RevocationRegistry> = issued_rev_status_list.into();
```

This does not work here?

Reply:
compiler complains I think due to `unwrap()`, which we need for later.

Reply:
Ah yeah when using unwrap it cannot infer that into should go to an `Option<T>`. Makes sense